packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -589,6 +589,10 @@ const AccountSwitcher = props => {
         canOpenMulti() &&
         (!props.show_eu_related_content || (props.show_eu_related_content && props.can_change_fiat_currency));
 
+    const can_manage_account_when_virtual =
+        props.is_virtual &&
+        (!props.show_eu_related_content || (props.show_eu_related_content && props.can_change_fiat_currency));
+
```

Comment:
@sergei-deriv can we check if  this works when we go to dtrader or dmt5 directly and not from traders hub. I remember i get undefined for `show_eu_related_content` when traders hub store was not initialized.

Reply:
I've rechecked and these parameters are not undefined when you open dtrader directly or from traders hub.